cms/server/contest/templates/documentation.html

```diff
@@ -23,6 +23,12 @@ <h3>Java</h3>
 <p>{% trans %}The main Java class of the solution should have exactly the same name as the task.{% endtrans %}</p>
 {% endif %}
 
+{% if contest.languages|map("to_language")|map(attribute="source_extensions")|any("contains", ".py") %}
```

Comment:
Need to check whether the documentation path is set. Otherwise, this may render a broken link.

Reply:
Exposed the config state in handler/template, seemed to work as expected.